Automattic/wp-calypso
# Add/plan selection design

#### Proposed Changes

* Updates the SensePlan step design to look like in the designs.
* Adds a basic error handling for SenseiPlan step.

#### Testing Instructions



* Start Sensei Tailored Flow
* Get to the Plan Selectoin Step.
* Confirm that it looks according to the design. km0MaMdbZMb5ySnGP2RSxg-fi-20021306%3A34108
* Confirm that it looks according to the design for smaller screens. km0MaMdbZMb5ySnGP2RSxg-fi-20021480%3A46366
* Disable your network and select a plan.
* Confirm that it has basic error handling.

#### Pre-merge Checklist






Related to [#Automattic/sensei-pro/1868](https://github.com/Automattic/sensei-pro/issues/1868)
